README.md

```diff
@@ -105,11 +114,7 @@ a free API for retrieving currency exchange information.
 
 ## Batch processing
```

Comment:
From what I understand, this is a third standalone task that is unrelated to the GET and POST parts, right? 

As it stands now, it's not very clear to me that this is the case. Perhaps structure the other parts with numbers:

1) GET Requests
2) POST Requests
3) Batch Processing